src/net.h

```diff
@@ -713,6 +707,12 @@ class CNode
 
     std::string ConnectionTypeAsString() const { return ::ConnectionTypeAsString(m_conn_type); }
 
+    //* We have completed a successful ping-pong keepalive. Update latest and best ping times. */
```

Comment:
I appreciate the added documentation 🙏 

nit
```suggestion
    /** We have completed a successful ping-pong keepalive. Update latest and best ping times. */
```

Reply:
:facepalm: I've fixed the doxygen style and also tweaked the comment wording a bit.